CONTRIBUTING.MD

```diff
@@ -25,24 +33,33 @@ You can contribute in many ways: openning or participating of a issue, reporting
 
     Note, using a `git add .` will automatically add all files. You can do a `git status` to see your changes, but do it before `git add`.
 
-6.  Commit your changes using a descriptive commit message.
-
+6.  Commit your changes using [commitizen](http://commitizen.github.io/cz-cli/).
+ (remember, before of to use commitizen, the packages should be installed previously)
     ```sh
-    git commit -m "Brief Description of Commit"
+    yarn commitizen
     ```
-7.  Push your commits to your Fork:
+
+7.  You'll be prompted to fill in any required fields and your commit messages will be formatted according to the standards defined by project.
+    
+    - Select the type of change that you're commiting (required)
+    - What is the scope of this change (e.g. component or file name)?
+    - Write a short, imperative tense description of the change (required)
+    - Provide a longer description of the change
+    - Are they any breaking changes? (y/N)
+    - Does this change affect any open issues? (y/N)
+
+8.  Push your commits to your Fork:
 
     ```sh
     git push -u origin branch-name
     ```
-8.  Submit a pull request.
+9.  Submit a pull request.
 
-## Submitting a Merge Request
+## Submitting a Pull Request
 
 Open your pull request, if you don't know how do it, check this [tutorial](https://docs.gitlab.com/ee/gitlab-basics/add-merge-request.html/)
```

Comment:
we are using a gitlab url here. use a github one.
(i did updated in catho-components a same url)